cdap-ui/app/directives/widget-container/widget-js-editor/widget-js-editor.js

```diff
@@ -31,9 +31,14 @@ angular.module(PKG.name + '.commons')
           workerPath: '/assets/bundle/ace-editor-worker-scripts',
           mode: $scope.mode || 'javascript',
           useWrapMode: true,
-          newLineMode: 'unix'
+          newLineMode: 'unix',
+          onLoad: aceLoaded
```

Comment:
I think we can do this (as specified by a workaround in angular ui ace editor [Workaround](https://github.com/angular-ui/ui-ace/blob/master/src/ui-ace.js#L100) 

```
$scope.aceoptions = {
   ...,
   newLineMode: 'unix',
   advanced: {
     tabSize: 2
   }
}
```